src/templates/finops-hub/modules/storage.bicep

```diff
@@ -76,9 +90,181 @@ resource storageAccount 'Microsoft.Storage/storageAccounts@2022-09-01' = {
   tags: union(tags, contains(tagsByResource, 'Microsoft.Storage/storageAccounts') ? tagsByResource['Microsoft.Storage/storageAccounts'] : {})
   properties: {
     supportsHttpsTrafficOnly: true
+    allowSharedKeyAccess: true
     isHnsEnabled: true
     minimumTlsVersion: 'TLS1_2'
     allowBlobPublicAccess: false
+    publicNetworkAccess: 'Enabled'
+    networkAcls: {
+      bypass: 'AzureServices'
+      defaultAction: enablePublicAccess ? 'Allow' : 'Deny'
+    }
+  }
+}
+
+resource scriptStorageAccount 'Microsoft.Storage/storageAccounts@2022-09-01' =  {
+  name: scriptStorageAccountName
+  location: location
+  sku: {
+    name: 'Standard_LRS' //sku
+  }
+  kind: 'StorageV2'// 'BlockBlobStorage'
+  tags: union(tags, contains(tagsByResource, 'Microsoft.Storage/storageAccounts') ? tagsByResource['Microsoft.Storage/storageAccounts'] : {})
+  properties: {
+    supportsHttpsTrafficOnly: true
+    allowSharedKeyAccess: true
+    isHnsEnabled: false
+    minimumTlsVersion: 'TLS1_2'
+    allowBlobPublicAccess: false
+    publicNetworkAccess: 'Enabled'
+    networkAcls: {
+      bypass: 'AzureServices'
+      defaultAction: 'Deny'
```

Comment:
This won't block deployments, right? Again, assuming not, but just want to confirm.

Reply:
Yes.  This is the middle checkbox option on the storage firewall.  It blocks all access except from the vNet we're deploying.